internal/proxy/oauthproxy_test.go

```diff
@@ -1336,3 +1321,78 @@ func TestHTTPSRedirect(t *testing.T) {
 		})
 	}
 }
+
+// We want to test that allowed email addresses passed in
+// via the upstream config will override those in the environment.
+//func TestAllowedEmailAddressOverrides(t *testing.T) {
+//	testCases := []struct {
+//		name                      string
+//		environmentEmailAddresses string
+//		upstreamEmailAddresses    []string
+//		expectedStatusCode        int
+//	}{
+//		{
+//			name:                      "environmentEmailAddress set, upstreamEmailAddress unset",
```

Comment:
Need to check this test out

Reply:
This test verified override behavior from the default configuration to the resolved state, ensuring that the defaults were set correctly and overridden by upstream specific configuration.

Reply:
This was added and accidentally pulled over from a separate branch -- it's a useful test to have, but not very relevant to this PR. I've removed it and will work on testing + re-adding it separately.